simulator/src/main/java/com/hazelcast/simulator/test/TestPhase.java

```diff
@@ -25,6 +25,9 @@
     LOCAL_WARMUP("local warmup", false),
     GLOBAL_WARMUP("global warmup", true),
     RUN("run", false),
+    WARMUP("warmup", false),
```

Comment:
Those phases should be placed before `RUN` to reflect the order of execution, shouldn't they?